gratipay/models/team.py

```diff
@@ -158,19 +158,6 @@ def to_dict(self):
         }
 
     def migrate_tips(self):
-        payment_instructions = self.db.all("""
-            SELECT pi.*
-              FROM payment_instructions pi
-              JOIN teams t ON t.slug = pi.team
-              JOIN participants p ON t.owner = p.username
-             WHERE p.username = %s
-               AND pi.ctime < t.ctime
-        """, (self.owner, ))
-
```

Comment:
Oh I'm sorry - I was wrong here. The `JOIN` statement accounts for subscriptions for all teams a user owns. 

If a user creates Team A, we migrate tips and then he creates Team B - we don't want to migrate tips again.

@aandis - Can you fix this and also add a test to ensure that this doesn't happen again?